ONSdigital/eq-publisher-v3
# Ear 1618 readable urls

https://jira.ons.gov.uk/browse/EAR-1618

How to review -

- Create a questionnaire with many pages - Question, Calculated Summary, List Collector etc.
- Enter unique page descriptions for the section and all the question pages.
- View the survey and answer all the question pages - check that the browser URL contains the page description entered - for example, if the page description is 'Summary page', then the URL will contain 'summary-page'.
- Similarly, for list collector questions, check that the URL contains the corresponding page description - for example, if the collection question page description is 'Collect page', the URL will contain 'add-block-collect-page'.
- In summary, for all the page URLs, the 'uuid' will not be displayed and will be replaced with the unique page descriptions.
- Add routing logic on some questions and check if the survey launches as expected.
- Launch as many surveys as possible and check if everything works as expected.
